server/db/driver/pg/orders.go

```diff
@@ -7,105 +7,55 @@ import (
 	"context"
 	"database/sql"
 	"database/sql/driver"
-	"encoding/hex"
 	"fmt"
-	"strconv"
-	"strings"
 
 	"decred.org/dcrdex/dex"
 	"decred.org/dcrdex/dex/order"
 	"decred.org/dcrdex/server/account"
 	"decred.org/dcrdex/server/db"
 	"decred.org/dcrdex/server/db/driver/pg/internal"
-	"github.com/lib/pq"
 )
 
-// utxo implements order.Outpoint
-type utxo struct {
-	txHash []byte
-	vout   uint32
-}
-
-func (u *utxo) TxHash() []byte { return u.txHash }
-func (u *utxo) Vout() uint32   { return u.vout }
-
-func newUtxo(txid string, vout uint32) *utxo {
-	hash, err := hex.DecodeString(txid)
-	if err != nil {
-		panic(err)
-	}
-	return &utxo{hash, vout}
-}
+// Wrap the CoinID slice to implement custom Scanner and Valuer.
+type dbCoins []order.CoinID
 
-func newUtxoFromOutpoint(outpoint string) (*utxo, error) {
-	outParts := strings.Split(outpoint, ":")
-	if len(outParts) != 2 {
-		return nil, fmt.Errorf("invalid outpoint %s", outpoint)
-	}
-	hash, err := hex.DecodeString(outParts[0])
-	if err != nil {
-		panic(err)
+// Value implements the sql/driver.Valuer interface.
+func (coins dbCoins) Value() (driver.Value, error) {
+	if len(coins) == 0 {
+		return []byte{}, nil
 	}
-	vout, err := strconv.ParseUint(outParts[1], 10, 32)
-	if err != nil {
-		return nil, fmt.Errorf("invalid vout %s: %v", outParts[1], err)
+	b := make([]byte, 0, len(coins)*(len(coins[0])+1))
+	for _, coin := range coins {
+		b = append(b, byte(len(coin)))
```

Comment:
I like this encoding.  However, `len(coins[0])` implies that each `len(coin)` is the same for each coin.  Is that the implication, or is this just a best guess total length and each `coin` is allowed to be a different length?

Reply:
Just a best guess. I'll doc it.

Reply:
It's estimate is probably accurate for all coins expected in initial release, but I think it will behave pretty well regardless of whether they are all exactly the same length.